tests/Unit/Framework/Transfer/AbstractTransferTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GacelaTest\Unit\Framework\Transfer;
+
+use Gacela\Framework\Transfer\UnknownPropertyException;
+use GacelaTest\Unit\Framework\Transfer\Fixtures\PersonTransfer;
+use PHPUnit\Framework\TestCase;
+
+final class AbstractTransferTest extends TestCase
+{
+    public function test_fluent_setters(): void
```

Comment:
Both tests are doing the same (`test_fluent_setters()` + `test_getters()`), but one is setting the parameters fluently and the other not.
It doesn't make sense because both methods are `setting` & `getting` the information for the assertion.

I would suggest to change the test name to something like `test_properties_transfer_object_fluently` & `test_properties_object_transfer` maybe?